container-core/src/main/java/com/yahoo/metrics/ContainerMetrics.java

```diff
@@ -10,7 +10,22 @@ public enum ContainerMetrics {
     HTTP_STATUS_3XX("http.status.3xx", Unit.RESPONSE, "Number of responses with a 3xx status"),
     HTTP_STATUS_4XX("http.status.4xx", Unit.RESPONSE, "Number of responses with a 4xx status"),
     HTTP_STATUS_5XX("http.status.5xx", Unit.RESPONSE, "Number of responses with a 5xx status"),
-    JDISC_GC_MS("jdisc.gc.ms", Unit.MILLISECOND, "Time spent in garbage collection");
+    JDISC_GC_MS("jdisc.gc.ms", Unit.MILLISECOND, "Time spent in garbage collection"),
+
+    JETTY_THREADPOOL_MAX_THREADS("jdisc.http.jetty.threadpool.thread.max", Unit.RESPONSE,
+            "Configured maximum number of threads"),
+    JETTY_THREADPOOL_MIN_THREADS("jdisc.http.jetty.threadpool.thread.min", Unit.RESPONSE,
+            "Configured minimum number of threads"),
+    JETTY_THREADPOOL_RESERVED_THREADS("jdisc.http.jetty.threadpool.thread.reserved", Unit.RESPONSE,
+            "Configured number of reserved threads or -1 for heuristic"),
+    JETTY_THREADPOOL_BUSY_THREADS("jdisc.http.jetty.threadpool.thread.busy", Unit.RESPONSE,
+            "Number of threads executing internal and transient jobs"),
+    JETTY_THREADPOOL_IDLE_THREADS("jdisc.http.jetty.threadpool.thread.idle", Unit.RESPONSE,
+            "Number of idle threads"),
+    JETTY_THREADPOOL_TOTAL_THREADS("jdisc.http.jetty.threadpool.thread.total", Unit.RESPONSE,
+            "Current number of threads"),
+    JETTY_THREADPOOL_QUEUE_SIZE("jdisc.http.jetty.threadpool.queue.size", Unit.RESPONSE,
+            "Current size of the job queue");
```

Comment:
All these should probably use `Unit.THREAD`

Reply:
all right, I was pondering on that - hang on ...